init.c

```diff
@@ -80,7 +80,6 @@ void init(int RestartSnapNum)
         P[i].Ti_drift = P[i].Ti_kick = All.Ti_Current;
 
         P[i].IsGarbage = 0;
-        P[i].IsNewParticle = 0;
```

Comment:
I am not sure about this. The okamoto feedback model is applied only to the newly formed star particles. Without the IsNewParticle flag, how do you ensure the semantics is still correct?